docs/source/examples/pt_zne.md

```diff
@@ -0,0 +1,386 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+```{tags} cirq, zne, pt, intermediate
+```
+# Zero-Noise Extrapolation with Pauli Twirling
+
+This tutorial explores how noise tailoring can improve the effectiveness of quantum error mitigation techniques.
+Specifically, we analyze how converting coherent noise into incoherent noise through Pauli Twirling (PT) impacts the performance of [Zero-Noise Extrapolation](../guide/zne.md) (ZNE).
+
+In this tutorial, we will:
+
+1. Define and compare coherent and incoherent noise
+2. Apply Pauli Twirling to transform coherent noise into incoherent noise
+3. Compare the performance of ZNE
+    1. on its own, and
+    2. in combination with [Pauli Twirling](../guide/pt.md)
+
+By the end of the example, you will understand when and how noise tailoring can enhance ZNE.
+
+## Coherent noise vs. Incoherent noise
+
+Noise on quantum devices can broadly categorized into two types: Coherent, Incoherent. Each has different properties that can be unfavorable toward a quantum circuit in different ways.
+
+**Coherent noise** is a reversible process as long as the noisy unitary transformation is known beforehand which is not always the case. These types of noise maintain the purity of the state. But in a quantum circuit subjected to coherent noise, the errors are easily carried across the circuit. This can be discerned through the **average gate infidelity** $r(\mathcal{E})$. When coherent errors contribute to a portion of the total error-rate, the worst case infidelity can scale as $\sqrt{r(\mathcal{E})}$ {cite}`Wallman_2014` which in turn reduces the performance of a quantum device by orders of magnitude. Thus, dealing with coherent noise requires a large resource overhead to acquire inferred knowledge of the noise unitaries which can then be used to reverse the effects.
+
+```{note}
+If $\mathcal{F}$ is the fidelity defining the success of preparing an arbitrary pure state $\rho$, then
+$1-\mathcal{F}$ is the **average gate infidelity**.
+```
```

Comment:
I like that we're bringing in some mathematical results here, but I do wonder if there is some ambiguity around which fidelity these statements are in reference to. Is this a process fidelity or state fidelity? Is $\mathcal{F}$ equal to what's defined here https://docs.quantum.ibm.com/api/qiskit/0.19/qiskit.quantum_info.average_gate_fidelity?

Reply:
It's the qiskit one.

Reply:
Then I think it should read

> If $\mathcal{F}$ is the average gate fidelity ...

WDYT? Should we add references here?